2023-12-18 13:55:15,122 [WARNING] PyNaCl is not installed, voice will NOT be supported
2023-12-18 13:55:15,128 [INFO] logging in using static token
2023-12-18 13:55:16,436 [INFO] Shard ID None has connected to Gateway (Session ID: f073b34ae953faa9a44ecf5482d5ba79).
2023-12-18 13:55:44,208 [INFO] 'execute_remove_member' called by Name:mazedias | ID:273479261308846110 from Server:Bester Server EU West | ID:770293700211114004
2023-12-18 13:56:12,883 [INFO] 'execute_remove_member' called by Name:mazedias | ID:273479261308846110 from Server:Bester Server EU West | ID:770293700211114004
2023-12-18 13:56:13,691 [ERROR] 'execute_save_event' failed with error 
2023-12-18 13:58:46,946 [WARNING] PyNaCl is not installed, voice will NOT be supported
2023-12-18 13:58:46,953 [INFO] logging in using static token
2023-12-18 13:58:48,394 [INFO] Shard ID None has connected to Gateway (Session ID: 1ddafbfa9f7d43b6164fcebe500e1984).
2023-12-18 13:58:59,250 [INFO] 'execute_remove_member' called by Name:mazedias | ID:273479261308846110 from Server:Bester Server EU West | ID:770293700211114004
2023-12-18 13:59:00,074 [ERROR] 'execute_save_event' failed because no member 'Mazedias' was found to remove
2023-12-18 13:59:24,160 [INFO] execute_link_member' called by Name:mazedias | ID:273479261308846110 from Server:Bester Server EU West | ID:770293700211114004
2023-12-18 14:03:27,042 [WARNING] PyNaCl is not installed, voice will NOT be supported
2023-12-18 14:03:27,050 [INFO] logging in using static token
2023-12-18 14:03:28,279 [INFO] Shard ID None has connected to Gateway (Session ID: 9e90ab39777438f56e6377f374c83faf).
2023-12-18 14:03:32,385 [INFO] execute_add_competition' called by Name:mazedias | ID:273479261308846110 from Server:Bester Server EU West | ID:770293700211114004
2023-12-18 14:03:39,322 [INFO] execute_add_competition' called by Name:mazedias | ID:273479261308846110 from Server:Bester Server EU West | ID:770293700211114004
2023-12-18 14:03:46,914 [INFO] execute_add_competition' called by Name:mazedias | ID:273479261308846110 from Server:Bester Server EU West | ID:770293700211114004
2023-12-18 14:15:38,232 [WARNING] PyNaCl is not installed, voice will NOT be supported
2023-12-18 14:15:38,238 [INFO] logging in using static token
2023-12-18 14:15:39,345 [INFO] Shard ID None has connected to Gateway (Session ID: 9efadb2f991f252e8c582748e04713e5).
2023-12-18 23:48:56,455 [WARNING] PyNaCl is not installed, voice will NOT be supported
2023-12-18 23:48:56,462 [INFO] logging in using static token
2023-12-18 23:48:57,720 [INFO] Shard ID None has connected to Gateway (Session ID: e219e4545f05368b2c141dfb2a735fae).
2023-12-18 23:49:20,469 [ERROR] Ignoring exception in command 'botstatus'
Traceback (most recent call last):
  File "/home/mazedias/.local/lib/python3.11/site-packages/discord/app_commands/commands.py", line 828, in _do_call
    return await self._callback(interaction, **params)  # type: ignore
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/mazedias/Documents/Projects/DT-Discord-Bot/new_src/main.py", line 154, in botstatus
    await execute_botstatus(interaction)
  File "/home/mazedias/Documents/Projects/DT-Discord-Bot/new_src/routines/setting_commands.py", line 29, in execute_botstatus
    players: list[Player] = load_players()
                            ^^^^^^^^^^^^^^
  File "/home/mazedias/Documents/Projects/DT-Discord-Bot/new_src/storage/storage_api.py", line 66, in load_players
    name = player.keys()[0]
           ~~~~~~~~~~~~~^^^
TypeError: 'dict_keys' object is not subscriptable

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "/home/mazedias/.local/lib/python3.11/site-packages/discord/app_commands/tree.py", line 1248, in _call
    await command._invoke_with_namespace(interaction, namespace)
  File "/home/mazedias/.local/lib/python3.11/site-packages/discord/app_commands/commands.py", line 853, in _invoke_with_namespace
    return await self._do_call(interaction, transformed_values)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/mazedias/.local/lib/python3.11/site-packages/discord/app_commands/commands.py", line 842, in _do_call
    raise CommandInvokeError(self, e) from e
discord.app_commands.errors.CommandInvokeError: Command 'botstatus' raised an exception: TypeError: 'dict_keys' object is not subscriptable
2023-12-18 23:53:59,632 [WARNING] PyNaCl is not installed, voice will NOT be supported
2023-12-18 23:53:59,640 [INFO] logging in using static token
2023-12-18 23:54:00,513 [INFO] Shard ID None has connected to Gateway (Session ID: c44e45163a4a6c69a461e6adcb75d8e7).
2023-12-18 23:54:04,332 [ERROR] Ignoring exception in command 'botstatus'
Traceback (most recent call last):
  File "/home/mazedias/.local/lib/python3.11/site-packages/discord/app_commands/commands.py", line 828, in _do_call
    return await self._callback(interaction, **params)  # type: ignore
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/mazedias/Documents/Projects/DT-Discord-Bot/new_src/main.py", line 154, in botstatus
    await execute_botstatus(interaction)
  File "/home/mazedias/Documents/Projects/DT-Discord-Bot/new_src/routines/setting_commands.py", line 29, in execute_botstatus
    players: list[Player] = load_players()
                            ^^^^^^^^^^^^^^
  File "/home/mazedias/Documents/Projects/DT-Discord-Bot/new_src/storage/storage_api.py", line 67, in load_players
    tag = player.get(id)[0]
          ~~~~~~~~~~~~~~^^^
TypeError: 'NoneType' object is not subscriptable

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "/home/mazedias/.local/lib/python3.11/site-packages/discord/app_commands/tree.py", line 1248, in _call
    await command._invoke_with_namespace(interaction, namespace)
  File "/home/mazedias/.local/lib/python3.11/site-packages/discord/app_commands/commands.py", line 853, in _invoke_with_namespace
    return await self._do_call(interaction, transformed_values)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/mazedias/.local/lib/python3.11/site-packages/discord/app_commands/commands.py", line 842, in _do_call
    raise CommandInvokeError(self, e) from e
discord.app_commands.errors.CommandInvokeError: Command 'botstatus' raised an exception: TypeError: 'NoneType' object is not subscriptable
2023-12-18 23:54:32,644 [WARNING] PyNaCl is not installed, voice will NOT be supported
2023-12-18 23:54:32,650 [INFO] logging in using static token
2023-12-18 23:54:34,256 [INFO] Shard ID None has connected to Gateway (Session ID: 4f3008179c8c71a0583f9a106ad1e076).
2023-12-18 23:54:36,824 [ERROR] Ignoring exception in command 'botstatus'
Traceback (most recent call last):
  File "/home/mazedias/.local/lib/python3.11/site-packages/discord/app_commands/commands.py", line 828, in _do_call
    return await self._callback(interaction, **params)  # type: ignore
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/mazedias/Documents/Projects/DT-Discord-Bot/new_src/main.py", line 154, in botstatus
    await execute_botstatus(interaction)
  File "/home/mazedias/Documents/Projects/DT-Discord-Bot/new_src/routines/setting_commands.py", line 29, in execute_botstatus
    players: list[Player] = load_players()
                            ^^^^^^^^^^^^^^
  File "/home/mazedias/Documents/Projects/DT-Discord-Bot/new_src/storage/storage_api.py", line 68, in load_players
    tag = player.get(id)[0]
          ~~~~~~~~~~~~~~^^^
TypeError: 'NoneType' object is not subscriptable

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "/home/mazedias/.local/lib/python3.11/site-packages/discord/app_commands/tree.py", line 1248, in _call
    await command._invoke_with_namespace(interaction, namespace)
  File "/home/mazedias/.local/lib/python3.11/site-packages/discord/app_commands/commands.py", line 853, in _invoke_with_namespace
    return await self._do_call(interaction, transformed_values)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/mazedias/.local/lib/python3.11/site-packages/discord/app_commands/commands.py", line 842, in _do_call
    raise CommandInvokeError(self, e) from e
discord.app_commands.errors.CommandInvokeError: Command 'botstatus' raised an exception: TypeError: 'NoneType' object is not subscriptable
2023-12-19 00:04:37,318 [WARNING] PyNaCl is not installed, voice will NOT be supported
2023-12-19 00:04:37,325 [INFO] logging in using static token
2023-12-19 00:04:38,731 [INFO] Shard ID None has connected to Gateway (Session ID: 76b3c29959bdec15e579c1fcb14802a3).
2023-12-19 00:04:42,891 [ERROR] Ignoring exception in command 'botstatus'
Traceback (most recent call last):
  File "/home/mazedias/.local/lib/python3.11/site-packages/discord/app_commands/commands.py", line 828, in _do_call
    return await self._callback(interaction, **params)  # type: ignore
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/mazedias/Documents/Projects/DT-Discord-Bot/new_src/main.py", line 154, in botstatus
    await execute_botstatus(interaction)
  File "/home/mazedias/Documents/Projects/DT-Discord-Bot/new_src/routines/setting_commands.py", line 31, in execute_botstatus
    ingame_name = get_players()
                  ^^^^^^^^^^^^^
TypeError: get_players() missing 1 required positional argument: 'guild_id'

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "/home/mazedias/.local/lib/python3.11/site-packages/discord/app_commands/tree.py", line 1248, in _call
    await command._invoke_with_namespace(interaction, namespace)
  File "/home/mazedias/.local/lib/python3.11/site-packages/discord/app_commands/commands.py", line 853, in _invoke_with_namespace
    return await self._do_call(interaction, transformed_values)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/mazedias/.local/lib/python3.11/site-packages/discord/app_commands/commands.py", line 842, in _do_call
    raise CommandInvokeError(self, e) from e
discord.app_commands.errors.CommandInvokeError: Command 'botstatus' raised an exception: TypeError: get_players() missing 1 required positional argument: 'guild_id'
2023-12-19 00:05:28,385 [WARNING] PyNaCl is not installed, voice will NOT be supported
2023-12-19 00:05:28,393 [INFO] logging in using static token
2023-12-19 00:05:29,473 [INFO] Shard ID None has connected to Gateway (Session ID: f75efde617c7f1354fc25fb4df0163ee).
2023-12-19 00:07:12,444 [WARNING] PyNaCl is not installed, voice will NOT be supported
2023-12-19 00:07:12,451 [INFO] logging in using static token
2023-12-19 00:07:13,764 [INFO] Shard ID None has connected to Gateway (Session ID: ec416a9daa8cbd36a512f5c9b191baa1).
2023-12-19 00:21:11,403 [WARNING] PyNaCl is not installed, voice will NOT be supported
2023-12-19 00:21:11,408 [INFO] logging in using static token
2023-12-19 00:21:12,666 [INFO] Shard ID None has connected to Gateway (Session ID: 4cc57c7415c8686fd79a5331e66da7a7).
2023-12-19 00:22:53,713 [WARNING] PyNaCl is not installed, voice will NOT be supported
2023-12-19 00:22:53,720 [INFO] logging in using static token
2023-12-19 00:22:55,001 [INFO] Shard ID None has connected to Gateway (Session ID: 838ab978c431ebe3693096a4d5bf54ae).
2023-12-19 00:23:03,501 [INFO] execute_link_member' called by Name:mazedias | ID:273479261308846110 from Server:Bester Server EU West | ID:770293700211114004
2023-12-19 00:23:10,823 [ERROR] Ignoring exception in command 'botstatus'
Traceback (most recent call last):
  File "/home/mazedias/.local/lib/python3.11/site-packages/discord/app_commands/commands.py", line 828, in _do_call
    return await self._callback(interaction, **params)  # type: ignore
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/mazedias/Documents/Projects/DT-Discord-Bot/new_src/main.py", line 154, in botstatus
    await execute_botstatus(interaction)
  File "/home/mazedias/Documents/Projects/DT-Discord-Bot/new_src/routines/setting_commands.py", line 38, in execute_botstatus
    member_ingame_names.remove(p.ingame_name)
ValueError: list.remove(x): x not in list

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "/home/mazedias/.local/lib/python3.11/site-packages/discord/app_commands/tree.py", line 1248, in _call
    await command._invoke_with_namespace(interaction, namespace)
  File "/home/mazedias/.local/lib/python3.11/site-packages/discord/app_commands/commands.py", line 853, in _invoke_with_namespace
    return await self._do_call(interaction, transformed_values)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/mazedias/.local/lib/python3.11/site-packages/discord/app_commands/commands.py", line 846, in _do_call
    raise CommandInvokeError(self, e) from e
discord.app_commands.errors.CommandInvokeError: Command 'botstatus' raised an exception: ValueError: list.remove(x): x not in list
2023-12-19 00:23:26,537 [WARNING] PyNaCl is not installed, voice will NOT be supported
2023-12-19 00:23:26,544 [INFO] logging in using static token
2023-12-19 00:23:27,810 [INFO] Shard ID None has connected to Gateway (Session ID: 0a9f315f2ef203cdd016018fd07d4e68).
2023-12-19 00:24:00,578 [WARNING] PyNaCl is not installed, voice will NOT be supported
2023-12-19 00:24:00,584 [INFO] logging in using static token
2023-12-19 00:24:01,937 [INFO] Shard ID None has connected to Gateway (Session ID: 33243cd900e908ca4ab9459736c76f9c).
2023-12-19 00:55:36,274 [WARNING] PyNaCl is not installed, voice will NOT be supported
2023-12-19 00:55:36,281 [INFO] logging in using static token
2023-12-19 00:55:37,385 [INFO] Shard ID None has connected to Gateway (Session ID: 780a0d69f8d75d46f92d02dbb9182d7f).
2023-12-19 00:55:42,318 [INFO] execute_add_competition' called by Name:mazedias | ID:273479261308846110 from Server:Bester Server EU West | ID:770293700211114004
2023-12-19 00:56:11,340 [WARNING] PyNaCl is not installed, voice will NOT be supported
2023-12-19 00:56:11,347 [INFO] logging in using static token
2023-12-19 00:56:12,399 [INFO] Shard ID None has connected to Gateway (Session ID: 50d2b54a1f36d432a21393b69662f820).
2023-12-19 00:56:13,739 [INFO] execute_add_competition' called by Name:mazedias | ID:273479261308846110 from Server:Bester Server EU West | ID:770293700211114004
2023-12-19 01:19:31,423 [WARNING] PyNaCl is not installed, voice will NOT be supported
2023-12-19 01:19:31,430 [INFO] logging in using static token
2023-12-19 01:19:32,861 [INFO] Shard ID None has connected to Gateway (Session ID: 32453562b23a259cb3c15e4dcfe0d292).
2023-12-19 01:19:37,132 [INFO] 'execute_summary' called by Name:mazedias | ID:273479261308846110 from Server:Bester Server EU West | ID:770293700211114004
2023-12-19 01:19:37,796 [ERROR] 'execute_summary' failed with error: LookupError
2023-12-19 01:21:24,736 [WARNING] PyNaCl is not installed, voice will NOT be supported
2023-12-19 01:21:24,743 [INFO] logging in using static token
2023-12-19 01:21:25,912 [INFO] Shard ID None has connected to Gateway (Session ID: 4ca12308880f7e2e14bd6def638164c2).
2023-12-19 01:21:30,266 [INFO] 'execute_summary' called by Name:mazedias | ID:273479261308846110 from Server:Bester Server EU West | ID:770293700211114004
2023-12-19 01:22:43,911 [INFO] execute_link_member' called by Name:mazedias | ID:273479261308846110 from Server:Bester Server EU West | ID:770293700211114004
2023-12-19 01:22:52,540 [INFO] 'execute_summary' called by Name:mazedias | ID:273479261308846110 from Server:Bester Server EU West | ID:770293700211114004
2023-12-19 01:24:40,375 [WARNING] PyNaCl is not installed, voice will NOT be supported
2023-12-19 01:24:40,381 [INFO] logging in using static token
2023-12-19 01:24:41,526 [INFO] Shard ID None has connected to Gateway (Session ID: 191bb02f9d4db63377535bdc8a32bff1).
2023-12-19 01:24:46,423 [INFO] 'execute_summary' called by Name:mazedias | ID:273479261308846110 from Server:Bester Server EU West | ID:770293700211114004
2023-12-19 01:25:14,006 [WARNING] PyNaCl is not installed, voice will NOT be supported
2023-12-19 01:25:14,013 [INFO] logging in using static token
2023-12-19 01:25:20,511 [INFO] Shard ID None has connected to Gateway (Session ID: 63eb764bf65452660c5ea1143559cdbe).
2023-12-19 01:25:24,964 [INFO] 'execute_summary' called by Name:mazedias | ID:273479261308846110 from Server:Bester Server EU West | ID:770293700211114004
2023-12-19 01:26:23,860 [INFO] 'execute_temp_ping_avoid' called by Name:mazedias | ID:273479261308846110 from Server:Bester Server EU West | ID:770293700211114004
2023-12-19 01:27:19,382 [INFO] 'execute_ping_again' called by Name:mazedias | ID:273479261308846110 from Server:Bester Server EU West | ID:770293700211114004
2023-12-19 01:27:29,389 [INFO] 'execute_temp_ping_avoid' called by Name:mazedias | ID:273479261308846110 from Server:Bester Server EU West | ID:770293700211114004
2023-12-19 01:27:40,809 [INFO] 'execute_temp_ping_avoid' called by Name:mazedias | ID:273479261308846110 from Server:Bester Server EU West | ID:770293700211114004
2023-12-19 01:28:19,603 [WARNING] PyNaCl is not installed, voice will NOT be supported
2023-12-19 01:28:19,610 [INFO] logging in using static token
2023-12-19 01:28:21,118 [INFO] Shard ID None has connected to Gateway (Session ID: 8f86feecccad0e5bfd9413c8415191bb).
2023-12-19 01:28:30,713 [INFO] 'execute_temp_ping_avoid' called by Name:mazedias | ID:273479261308846110 from Server:Bester Server EU West | ID:770293700211114004
2023-12-19 01:28:48,350 [INFO] 'execute_ping_again' called by Name:mazedias | ID:273479261308846110 from Server:Bester Server EU West | ID:770293700211114004
2023-12-19 01:29:11,309 [INFO] execute_link_member' called by Name:mazedias | ID:273479261308846110 from Server:Bester Server EU West | ID:770293700211114004
2023-12-19 01:29:29,203 [INFO] 'execute_temp_ping_avoid' called by Name:mazedias | ID:273479261308846110 from Server:Bester Server EU West | ID:770293700211114004
2023-12-19 01:29:36,302 [INFO] 'execute_temp_ping_avoid' called by Name:mazedias | ID:273479261308846110 from Server:Bester Server EU West | ID:770293700211114004
2023-12-19 01:30:06,886 [INFO] 'execute_temp_ping_avoid' called by Name:mazedias | ID:273479261308846110 from Server:Bester Server EU West | ID:770293700211114004
2023-12-19 01:30:13,792 [WARNING] PyNaCl is not installed, voice will NOT be supported
2023-12-19 01:30:13,799 [INFO] logging in using static token
2023-12-19 01:30:15,001 [INFO] Shard ID None has connected to Gateway (Session ID: 13620aa5eca728e9660acd90550ddb1e).
2023-12-19 01:30:21,943 [INFO] 'execute_ping_again' called by Name:mazedias | ID:273479261308846110 from Server:Bester Server EU West | ID:770293700211114004
2023-12-19 01:30:27,692 [INFO] 'execute_ping_again' called by Name:mazedias | ID:273479261308846110 from Server:Bester Server EU West | ID:770293700211114004
2023-12-19 01:30:38,173 [INFO] 'execute_ping_again' called by Name:mazedias | ID:273479261308846110 from Server:Bester Server EU West | ID:770293700211114004
2023-12-19 01:30:52,126 [INFO] 'execute_perma_ping_avoid' called by Name:mazedias | ID:273479261308846110 from Server:Bester Server EU West | ID:770293700211114004
2023-12-19 01:31:20,644 [INFO] 'execute_eventdata' called by Name:mazedias | ID:273479261308846110 from Server:Bester Server EU West | ID:770293700211114004
2023-12-19 01:31:40,888 [INFO] 'execute_iteminfo' called by Name:mazedias | ID:273479261308846110 from Server:Bester Server EU West | ID:770293700211114004
2023-12-19 01:31:55,433 [INFO] 'execute_summary' called by Name:mazedias | ID:273479261308846110 from Server:Bester Server EU West | ID:770293700211114004
2023-12-19 01:32:05,875 [INFO] 'execute_remove_member' called by Name:mazedias | ID:273479261308846110 from Server:Bester Server EU West | ID:770293700211114004
2023-12-19 01:32:16,109 [INFO] 'execute_save_event' called by Name:mazedias | ID:273479261308846110 from Server:Bester Server EU West | ID:770293700211114004
2023-12-24 16:18:16,637 [WARNING] PyNaCl is not installed, voice will NOT be supported
2023-12-24 16:18:16,641 [INFO] logging in using static token
2023-12-24 16:18:19,152 [INFO] Shard ID None has connected to Gateway (Session ID: 0f027d9a16a972ebe4f91101b4d4c12d).
2023-12-24 16:19:17,988 [INFO] 'execute_new_event' called by Name:mazedias | ID:273479261308846110 from Server:Bester Server EU West | ID:770293700211114004
2023-12-24 16:20:14,886 [INFO] 'execute_iteminfo' called by Name:mazedias | ID:273479261308846110 from Server:Bester Server EU West | ID:770293700211114004
2023-12-24 16:20:24,199 [INFO] 'execute_iteminfo' called by Name:mazedias | ID:273479261308846110 from Server:Bester Server EU West | ID:770293700211114004
2023-12-24 16:20:43,641 [INFO] 'execute_iteminfo' called by Name:mazedias | ID:273479261308846110 from Server:Bester Server EU West | ID:770293700211114004
2023-12-24 16:21:15,980 [INFO] execute_add_competition' called by Name:mazedias | ID:273479261308846110 from Server:Bester Server EU West | ID:770293700211114004
2023-12-24 16:21:36,553 [INFO] execute_add_competition' called by Name:mazedias | ID:273479261308846110 from Server:Bester Server EU West | ID:770293700211114004
2023-12-24 16:22:05,932 [INFO] execute_add_competition' called by Name:mazedias | ID:273479261308846110 from Server:Bester Server EU West | ID:770293700211114004
2023-12-24 16:25:07,087 [WARNING] PyNaCl is not installed, voice will NOT be supported
2023-12-24 16:25:07,091 [INFO] logging in using static token
2023-12-24 16:25:09,112 [INFO] Shard ID None has connected to Gateway (Session ID: 86e24d9f042c277ffd3697600fcc5303).
2023-12-24 16:25:13,808 [INFO] execute_add_competition' called by Name:mazedias | ID:273479261308846110 from Server:Bester Server EU West | ID:770293700211114004
2023-12-24 16:25:14,762 [ERROR] 'execute_add_competition' failed because a guild with id 1 is already stored.
2023-12-24 16:25:21,170 [INFO] execute_add_competition' called by Name:mazedias | ID:273479261308846110 from Server:Bester Server EU West | ID:770293700211114004
2023-12-24 16:26:07,358 [WARNING] PyNaCl is not installed, voice will NOT be supported
2023-12-24 16:26:07,362 [INFO] logging in using static token
2023-12-24 16:26:08,544 [INFO] Shard ID None has connected to Gateway (Session ID: 01271d56de0d688710fd1b1d6eea1132).
2023-12-24 16:26:12,337 [INFO] execute_add_competition' called by Name:mazedias | ID:273479261308846110 from Server:Bester Server EU West | ID:770293700211114004
2023-12-24 16:26:35,570 [INFO] execute_link_member' called by Name:mazedias | ID:273479261308846110 from Server:Bester Server EU West | ID:770293700211114004
2023-12-24 16:26:39,251 [INFO] execute_add_competition' called by Name:mazedias | ID:273479261308846110 from Server:Bester Server EU West | ID:770293700211114004
2023-12-24 16:27:06,603 [INFO] 'execute_remove_member' called by Name:mazedias | ID:273479261308846110 from Server:Bester Server EU West | ID:770293700211114004
2023-12-24 16:27:42,494 [INFO] execute_link_member' called by Name:mazedias | ID:273479261308846110 from Server:Bester Server EU West | ID:770293700211114004
2023-12-24 16:27:48,334 [INFO] execute_add_competition' called by Name:mazedias | ID:273479261308846110 from Server:Bester Server EU West | ID:770293700211114004
2023-12-24 16:28:12,578 [INFO] 'execute_remove_member' called by Name:mazedias | ID:273479261308846110 from Server:Bester Server EU West | ID:770293700211114004
2023-12-24 16:28:16,573 [INFO] 'execute_remove_member' called by Name:mazedias | ID:273479261308846110 from Server:Bester Server EU West | ID:770293700211114004
2023-12-24 16:28:17,690 [ERROR] 'execute_save_event' failed because no member 'tom' was found to remove
2023-12-24 16:28:44,610 [INFO] 'execute_summary' called by Name:mazedias | ID:273479261308846110 from Server:Bester Server EU West | ID:770293700211114004
2023-12-24 16:28:52,763 [INFO] 'execute_summary' called by Name:mazedias | ID:273479261308846110 from Server:Bester Server EU West | ID:770293700211114004
2023-12-24 16:29:24,208 [INFO] execute_link_member' called by Name:mazedias | ID:273479261308846110 from Server:Bester Server EU West | ID:770293700211114004
2023-12-24 16:29:31,914 [INFO] 'execute_summary' called by Name:mazedias | ID:273479261308846110 from Server:Bester Server EU West | ID:770293700211114004
2023-12-24 16:29:52,865 [INFO] 'execute_temp_ping_avoid' called by Name:mazedias | ID:273479261308846110 from Server:Bester Server EU West | ID:770293700211114004
2023-12-24 16:29:56,755 [INFO] 'execute_summary' called by Name:mazedias | ID:273479261308846110 from Server:Bester Server EU West | ID:770293700211114004
2023-12-24 16:30:13,491 [INFO] 'execute_ping_again' called by Name:mazedias | ID:273479261308846110 from Server:Bester Server EU West | ID:770293700211114004
2023-12-24 16:30:16,609 [INFO] 'execute_summary' called by Name:mazedias | ID:273479261308846110 from Server:Bester Server EU West | ID:770293700211114004
2023-12-24 16:34:03,959 [WARNING] PyNaCl is not installed, voice will NOT be supported
2023-12-24 16:34:03,963 [INFO] logging in using static token
2023-12-24 16:34:05,762 [INFO] Shard ID None has connected to Gateway (Session ID: d52f2caf6417acc2c45ca9cf057d5820).
2023-12-24 16:34:15,246 [INFO] execute_link_member' called by Name:mazedias | ID:273479261308846110 from Server:Bester Server EU West | ID:770293700211114004
2023-12-24 16:34:24,766 [INFO] 'execute_temp_ping_avoid' called by Name:mazedias | ID:273479261308846110 from Server:Bester Server EU West | ID:770293700211114004
2023-12-24 16:34:36,201 [INFO] execute_link_member' called by Name:mazedias | ID:273479261308846110 from Server:Bester Server EU West | ID:770293700211114004
2023-12-24 16:34:43,393 [INFO] 'execute_temp_ping_avoid' called by Name:mazedias | ID:273479261308846110 from Server:Bester Server EU West | ID:770293700211114004
2023-12-24 16:34:47,398 [INFO] 'execute_save_event' called by Name:mazedias | ID:273479261308846110 from Server:Bester Server EU West | ID:770293700211114004
2023-12-24 16:35:08,460 [INFO] 'execute_temp_ping_avoid' called by Name:mazedias | ID:273479261308846110 from Server:Bester Server EU West | ID:770293700211114004
2023-12-24 16:35:16,036 [INFO] 'execute_temp_ping_avoid' called by Name:mazedias | ID:273479261308846110 from Server:Bester Server EU West | ID:770293700211114004
2023-12-24 16:35:25,241 [INFO] 'execute_perma_ping_avoid' called by Name:mazedias | ID:273479261308846110 from Server:Bester Server EU West | ID:770293700211114004
2023-12-24 16:35:33,332 [INFO] 'execute_perma_ping_avoid' called by Name:mazedias | ID:273479261308846110 from Server:Bester Server EU West | ID:770293700211114004
2023-12-24 16:35:38,552 [INFO] 'execute_save_event' called by Name:mazedias | ID:273479261308846110 from Server:Bester Server EU West | ID:770293700211114004
2023-12-24 16:35:49,089 [INFO] 'execute_summary' called by Name:mazedias | ID:273479261308846110 from Server:Bester Server EU West | ID:770293700211114004
2023-12-24 16:36:03,821 [INFO] 'execute_ping_again' called by Name:mazedias | ID:273479261308846110 from Server:Bester Server EU West | ID:770293700211114004
2023-12-24 16:36:08,033 [INFO] 'execute_save_event' called by Name:mazedias | ID:273479261308846110 from Server:Bester Server EU West | ID:770293700211114004
2023-12-24 16:37:41,719 [WARNING] PyNaCl is not installed, voice will NOT be supported
2023-12-24 16:37:41,724 [INFO] logging in using static token
2023-12-24 16:37:44,329 [INFO] Shard ID None has connected to Gateway (Session ID: 9153c5009d785edc08d8ff347eb9c13c).
2023-12-24 16:37:50,655 [INFO] 'execute_save_event' called by Name:mazedias | ID:273479261308846110 from Server:Bester Server EU West | ID:770293700211114004
2023-12-24 16:38:26,235 [INFO] 'execute_eventdata' called by Name:mazedias | ID:273479261308846110 from Server:Bester Server EU West | ID:770293700211114004
2023-12-24 16:38:28,245 [ERROR] Ignoring exception in command 'eventdata'
Traceback (most recent call last):
  File "/home/mazedias/.local/lib/python3.11/site-packages/discord/app_commands/commands.py", line 828, in _do_call
    return await self._callback(interaction, **params)  # type: ignore
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/mazedias/Documents/Projects/DT-Discord-Bot/new_src/main.py", line 45, in eventdata
    await execute_eventdata(interaction)
  File "/home/mazedias/Documents/Projects/DT-Discord-Bot/new_src/routines/event_commands.py", line 83, in execute_eventdata
    f"• Aktive Spieler: {get_active_players(guild.id)}\n• Durchschnittliche Spenden: {get_donation_mean(guild.id)}\n• Insgesamte Spenden: {get_total_donations(guild.id)}\n• Geschätze Runde: {predict_current_round(get_total_donations(guild.id), get_active_event())}"
                                                                                                                                                                                                                                                    ^^^^^^^^^^^^^^^^^^
  File "/home/mazedias/Documents/Projects/DT-Discord-Bot/new_src/storage/storage_api.py", line 178, in get_active_event
    raise LookupError("No active event found!")
LookupError: No active event found!

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "/home/mazedias/.local/lib/python3.11/site-packages/discord/app_commands/tree.py", line 1248, in _call
    await command._invoke_with_namespace(interaction, namespace)
  File "/home/mazedias/.local/lib/python3.11/site-packages/discord/app_commands/commands.py", line 853, in _invoke_with_namespace
    return await self._do_call(interaction, transformed_values)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/mazedias/.local/lib/python3.11/site-packages/discord/app_commands/commands.py", line 846, in _do_call
    raise CommandInvokeError(self, e) from e
discord.app_commands.errors.CommandInvokeError: Command 'eventdata' raised an exception: LookupError: No active event found!
2023-12-24 16:45:57,881 [INFO] 'execute_summary' called by Name:mazedias | ID:273479261308846110 from Server:Bester Server EU West | ID:770293700211114004
2023-12-24 16:46:17,235 [WARNING] PyNaCl is not installed, voice will NOT be supported
2023-12-24 16:46:17,241 [INFO] logging in using static token
2023-12-24 16:46:18,949 [INFO] Shard ID None has connected to Gateway (Session ID: b06f90f01d58bc32a212ca826556629f).
2023-12-24 16:46:26,155 [INFO] 'execute_eventdata' called by Name:mazedias | ID:273479261308846110 from Server:Bester Server EU West | ID:770293700211114004
2023-12-24 16:46:26,735 [ERROR] 'execute_eventdata' failed with error: LookupError since no active event was found
2023-12-24 16:46:35,544 [INFO] 'execute_eventdata' called by Name:mazedias | ID:273479261308846110 from Server:Bester Server EU West | ID:770293700211114004
2023-12-28 22:37:52,080 [WARNING] PyNaCl is not installed, voice will NOT be supported
2023-12-28 22:37:52,083 [INFO] logging in using static token
2023-12-28 22:37:54,000 [INFO] Shard ID None has connected to Gateway (Session ID: 2a58ab14c718315ef827e751e8fc76de).
2023-12-28 22:38:23,494 [INFO] 'execute_save_event' called by Name:mazedias | ID:273479261308846110 from Server:Bester Server EU West | ID:770293700211114004
2023-12-28 22:39:14,228 [INFO] 'execute_new_event' called by Name:mazedias | ID:273479261308846110 from Server:Bester Server EU West | ID:770293700211114004
2023-12-28 22:39:23,207 [INFO] 'execute_iteminfo' called by Name:mazedias | ID:273479261308846110 from Server:Bester Server EU West | ID:770293700211114004
2023-12-28 22:41:13,608 [WARNING] PyNaCl is not installed, voice will NOT be supported
2023-12-28 22:41:13,613 [INFO] logging in using static token
2023-12-28 22:41:16,038 [INFO] Shard ID None has connected to Gateway (Session ID: 13f19bb24ccf36eb600fae41ddb3e40a).
2023-12-28 22:41:22,729 [INFO] 'execute_iteminfo' called by Name:mazedias | ID:273479261308846110 from Server:Bester Server EU West | ID:770293700211114004
2023-12-28 22:50:00,232 [WARNING] PyNaCl is not installed, voice will NOT be supported
2023-12-28 22:50:00,236 [INFO] logging in using static token
2023-12-28 22:50:01,444 [INFO] Shard ID None has connected to Gateway (Session ID: 1dd39c600b785201ae2982afd43717f7).
2023-12-28 22:50:09,746 [INFO] 'execute_iteminfo' called by Name:mazedias | ID:273479261308846110 from Server:Bester Server EU West | ID:770293700211114004
2023-12-28 23:34:39,646 [WARNING] PyNaCl is not installed, voice will NOT be supported
2023-12-28 23:34:39,651 [INFO] logging in using static token
2023-12-28 23:34:40,642 [INFO] Shard ID None has connected to Gateway (Session ID: 8a8019afcdf2f75f65e6ff7f2e6e2760).
2023-12-28 23:34:56,227 [INFO] 'execute_iteminfo' called by Name:mazedias | ID:273479261308846110 from Server:Bester Server EU West | ID:770293700211114004
